MapboxNavigation/Dictionary.swift

```diff
@@ -14,3 +14,18 @@ extension Dictionary where Key == Int, Value: NSExpression {
         return newCameraStop as! Dictionary<Key, Value>
     }
 }
+
+extension Dictionary where Key == Int, Value == Double {
+    /**
+     Returns a copy of the stop dictionary with each value multiplied by the given factor.
+     */
+    public func multiplied(by factor: Double) -> Dictionary {
```

Comment:
Deze niet beter als internal?

Reply:
De variant op NSExpression is ook public.